### DISCUSSION TITLE
Rename-DbaDatabase different logical names

### DISCUSSION THREAD

**Discussion by rakyzoly** [dataplat/dbatools#9178] [Q&A]
Hi,

I am wondering if it is possible to rename a Database different logical name for Data and Log file.
Yes, with placeholders <FT> it is possible, but I need strict custom logical names, for example:
-DataBaseLogicalName
-DataBaseLogicalName_log
I didn't find a solution anywhere.

Thanks in advance

**Comment by benny779** [comment]
Is this feature on the roadmap?
